### DISCUSSION TITLE
Is it possible to add more icons

### DISCUSSION THREAD

**Discussion by RexS98** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#335] [Q&A] [answered]
From readme I saw Icon list entry options.
But not sure if can add more beyong the default 3 icons. (Start,locate,Fan speed)
I'd like to add stop, back to dock.  Not sure if this is existing function.

**Comment by PiotrMachowski** [comment, marked as answer]
They are there, but are hidden when vacuum is docked.

It is possible to add own icons, using `icons` option
